lib/resources/dstu2/appointment.yaml

```diff
@@ -47,9 +45,9 @@ fields:
 
 - name: slot
   required: 'Yes'
-  cardinality: 1..1
   type: Reference (Slot)
```

Comment:
List of Reference (Slot)

Reply:
[Updated](https://github.com/cerner/fhir.cerner.com/pull/374/commits/6fb0b4a5a1bdcd0b07e086f542d44ded83b12430?file-filters%5B%5D=.yaml#diff-dda110f273dfe3a3ddac84fe6dacc327L48)